.github/workflows/playwright.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint, typecheck, build, run and e2e test the CRGA
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Do we need to run the tests for pushes to the `main` branch? Typically all changes should be tested in the feature branch  (as suggested) before merged into it.

Reply:
This was also autogenerated, and I can live with your variant or the current one. Technically differences can appear for feature branches and the `main`-branch, as there everything converges. Tell me which variant you want, I'll adjust accordingly.

Reply:
I would favour to not run all the tests on the main branch again.